scripts/test-pipelines.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+shopt -s globstar
+shopt -s nullglob
+
+# Quote the heredoc to prevent shell expansion
+cat << "EOF"
+workflow:
+  rules:
+    # Disable merge request pipelines
+    - if: $CI_MERGE_REQUEST_ID
+      when: never
+    - when: always
```

Comment:
I'm not sure if these are needed.

But you should have newline separation between the different sections, like we currently do in js-polykey. Like `default:` should on another newline.

Reply:
Those things aren't inherited by child pipelines (I've noticed this with the `stages` section in particular), but I can add in newline separation.

Reply:
Yes, they are not inherited, but I'm not sure if child pipelines even need these lines at all, because job execution is determined by the pipeline yaml's generation.